pymodbus/repl/server/cli.py

```diff
@@ -17,13 +19,13 @@
 _logger = logging.getLogger(__name__)
 
 TITLE = (
```

Comment:
We need to get a better way of representing this banner. This still gets mangled up.